WBooks/LibraryViewController.swift

```diff
@@ -19,6 +19,8 @@ final class LibraryViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationItem.title = NSLocalizedString("NAVIGATION_BAR_TITLE_LIBRARY", comment: "")
+        tableView.separatorStyle = .none
+        tableView.backgroundColor = UIColor.init(red: 234, green: 246, blue: 250, alpha: 1)
```

Comment:
What we do normally is create a UIColor extension and add static variables to return the colors of the app. So for example you could call `UIColor.wBooksMainColor` or `UIColor.wBooksBackground`. 
An example from another project:
```
internal extension UIColor {
    
    static var istaliBlue: UIColor {
        return UIColor(colorLiteralRed: 49.0/255.0, green: 105.0/255.0, blue: 157.0/255.0, alpha: 1)
    }

}
```